ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml

```diff
@@ -161,7 +161,7 @@ periodics:
     job-impl.setup-pod: launch-osd-create-create
     job-impl.workload-container: test
     job-impl.workload-pod: launch-wait
-  cluster: build01
+  cluster: build02
```

Comment:
/hold

Please remove this change, it's a flake (being fixed already).

Reply:
will do, thanks!